content/tutorials/kit-assembly.md

```diff
@@ -2,27 +2,34 @@
 title: Connecting your kit
 ---
 
-## The bits you should have
+##Essential Hardware##
 - Raspberry Pi (the board in a plastic case with many USB ports)
 - Power Board (the board in a plastic case with many green sockets and two buttons)
 - Motor Board (the other board in a plastic case)
 - Arduino (The blue, exposed board with a metal USB-B connector)
 - Arduino Servo Shield (another blue board, hopefully on top of the first)
+If any of this is unsure to you check on the underside of the board it should be labelled 
```

Comment:
I've got no idea what this sentence says or what it's actually for?

Reply:
I think it's saying there's information on the back of the motor/power board that would help them?